### ISSUE TITLE
Azure support

### ISSUE THREAD

**Issue by elkorba** [sorry-cypress/sorry-cypress#74]
Hi,
are there any plans to support one of the Azure Storage services for screenshot driver?
If yes then when we can expect it?

Regards

**Comment by agoldis** [comment]
Currently there're no plans for Azure - if you have expertise / time contributions are appreciated!

**Comment by weyert** [comment]
The easiest way to accomplish this without code changes would be to put Minio in between as the S3 compatible API which then talks to Azure Storage: https://docs.min.io/docs/minio-gateway-for-azure.html

An alternative would be to write an abstraction so you can write ones for Azure Blob Storage, Google Blob Storage and one for the S3 compatible ones

**Comment by agoldis** [comment]
FYI minio driver is now available https://github.com/agoldis/sorry-cypress/releases/tag/v0.6.0

**Comment by onimougwo** [comment]
Still no plan for Azure support ? My company uses Azure Devops for CI, does that mean I can't use sorry-cypress ?

**Comment by agoldis** [comment]
@onimougwo of course you can. It's just a docker images that you can deploy on any cloud provider. Unfortunately I don't  use Azure personally so I didn't write documentation. May be @weyert can help?

**Comment by DanielleCogs** [comment]
@weyert any new ideas on Azure Storage use? Started going through the setup for MinIO Azure Gateway but looks like they will stop supporting it soon [(related MinIO issue)](https://github.com/minio/minio/issues/14331).

**Comment by weyert** [comment]
@DanielleCogs Sorry, I stopped using Cypress

**Comment by roldengarm** [comment]
What are the implications when running this on Azure? Setting up the Docker image is not a problem, but obviously I'd like to see the screenshots as well. With MinIO gone, is there an alternative?

**Comment by DanielleCogs** [comment]
Yes, setting up the Docker image is fine. I was trying to use the AWS Cloud Formation setup as a template to use the equivalent Azure resources, but without S3 compatibility and MinIO being deprecated, we weren't able to deploy.

**Comment by anilpujaraofficial** [comment]
how to give custome port for sorry cypress dashboard?
